MicroProfile-Fault-Tolerance/tck-runner/.mvn/extensions.xml

```diff
@@ -0,0 +1,11 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
+
+    <extension>
```

Comment:
Does this PR work properly if this is removed and the parent extension is updated?

Reply:
If I remove the extension from the module it does not work at all as if the extension is not inherited.

Reply:
That wasn't the behaviour I got at all. Is it possible that it is the .m2 repository after all?

Reply:
It might be that both `.m2` and `.mvn` work. I suspect this is the case from the behaviour I got during tests. Anyhow, the reason I got 1.1 profile activated was `payara.version` being `5.184` if not specified which correctly activated 1.1. So I think we are good now as it is.

Reply:
I believe if you update the `extensions.xml` in the project root and delete this file it will work. If you don't observe this, we can discuss it either over chat or a call whenever!

Reply:
Updated the root extension to `0.2` and removed the module extension. Now this worked. Don't know what I was doing wrong before.